pom.xml

```diff
@@ -259,6 +254,24 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <artifactId>maven-enforcer-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>display-info</id>
+            <configuration>
+              <rules>
+                <requireUpperBoundDeps>
+                  <excludes combine.children="append">
+                    <exclude>com.damnhandy:handy-uri-templates</exclude>
+                    <exclude>org.jenkins-ci.main:jenkins-test-harness</exclude>
```

Comment:
Perhaps better to introduce new version of parent pom and bom in separate PR?

Reply:
well I though it is required, to use the same minimum version as dependent plugins, otherwise the `gitlab-branch-cource` plugin will be made available to users of older jenkins without the new `gitlab-api` plugin.

Reply:
jenkins version != parent pom :)

Also 2.263.4 was released 2021-02-10 with plenty security issues. I'd hope more users are using a newer Jenkins version.

Reply:
Yeah, `gitlab-branch-source-plugin` is currently using plugin parent POM 4.1, which is from April 2020. That is a very old build toolchain. I'd suggest using something closer to when the minimum Jenkins version (i.e. 2.289.3) was released. [4.25](https://github.com/jenkinsci/plugin-pom/releases/tag/plugin-4.25) should be fairly easy to upgrade to. 4.26 had some breaking changes, so you may want to take one small bite at a time. :-)

Reply:
I filed #189 to try and make Daniel's life a little easier.

Reply:
Merged, @dcendents you should be able to rebase and remove these lines 😄